### DISCUSSION TITLE
Lists with images

### DISCUSSION THREAD

**Discussion by kamieniarz** [squidfunk/mkdocs-material#5299] [General]
I'd like to make lists work with images but all I can do is adjust everything to specific screen resolution. Any other configuration (different resolutions and actually the most important - mobile view) makes page look simply... ugly. Here's my example

Code
![image](https://user-images.githubusercontent.com/9063907/229292062-18ba5677-4cd4-45f7-8c6a-a5cfdc4c6da8.png)

Desktop - I made it look nice, red one is image, black one is text
![image](https://user-images.githubusercontent.com/9063907/229291502-8cc1c9e8-5048-4184-8c64-8c392604ad45.png)

Mobile - this is bad :/
![image](https://user-images.githubusercontent.com/9063907/229291627-b8e7f976-4acc-42b6-b470-945d2ea26e2c.png)
![image](https://user-images.githubusercontent.com/9063907/229291728-b8a67c7f-ae2b-4e4a-8fe8-3b2cc5de7dc1.png)

Maybe there's something I'm doing wrong? What are your sugestions?

PS obviously instead of list I could use sth like `:octicons-dot-fill-16:` so text would be more "felxible" but styling will be different - there won't be that small gap between each list element and also there won't be that tab in each line of the same element

**Comment by kamilkrzyskow** [comment]
Too much code hidden away tbh.
Create a minimal example with simple shapes as images and some lorem ipsum code.
And maybe even draw a layout of what you envision.

One possible issue is that you set the width=250 to these images.
This is a static 250px, which doesn't scale well for mobiles.
Use extra CSS to set a percentage based width and/or use media queries to set different layouts depending on the viewport.

Also maybe try to keep it simple and add the image in between list entries? :v:
Or maybe try putting the image in an inline admonition.

https://squidfunk.github.io/mkdocs-material/reference/admonitions/#inline-blocks
